proto/OptimizerConfig.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto2";
+ 
+option optimize_for = LITE_RUNTIME;
```

Comment:
I agree with helin, we should focus on having the first working implementation, there should be many optimizer work later